cmd/clusterctl/config/manifest/clusterctl-api.yaml

```diff
@@ -36,24 +36,33 @@ spec:
         description: Provider defines an entry in the provider inventory.
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
Change in this file seems out of scope.
Guessing if you have an old version of controller-tools (you can get rid of it with `make clean-bin`)

Reply:
Thanks for pointing that out - I've updated now.